composer.json

```diff
@@ -47,26 +47,27 @@
         "php": "~8.1.0 || ~8.2.0",
         "ext-dom": "*",
         "composer-plugin-api": "^2.2",
-        "azjezz/psl": "^2.3.0",
+        "azjezz/psl": "^2.4.1",
         "beberlei/assert": "^v3.3.2",
-        "composer/composer": "^2.4.4",
-        "doctrine/rst-parser": "^0.5.2",
-        "nikic/php-parser": "^v4.15.2",
-        "symfony/config": "^v6.2.0",
-        "symfony/console": "^v6.2.1",
-        "symfony/finder": "^v6.2.0",
-        "symfony/process": "^v6.2.0",
+        "composer/composer": "^2.5.2",
+        "doctrine/rst-parser": "^0.5.3",
+        "nikic/php-parser": "^v4.15.3",
+        "symfony/config": "^v6.2.5",
+        "symfony/console": "^v6.2.5",
+        "symfony/finder": "^v6.2.5",
+        "symfony/process": "^v6.2.5",
         "webmozart/assert": "^1.11.0"
     },
     "require-dev": {
-        "doctrine/coding-standard": "^11.0.0",
-        "infection/infection": "^0.26.16",
-        "maglnet/composer-require-checker": "^4.4.0",
+        "doctrine/coding-standard": "^11.1.0",
+        "infection/infection": "dev-master",
```

Comment:
We should actually use a tagged version instead.

Reply:
```suggestion
        "infection/infection": "^0.26.19",
```
👍🏻